test/provision_test.py

```diff
@@ -183,3 +183,45 @@ def test_for_multiple_dispenses_of_resource_in_containers_larger_than_900ml(self
             "split_provisions_by_volume.json"
         )
         assert expected_instruction_as_json == actual_instruction_as_json
+
+    def test_provision_well_with_mass(self):
+        p = Protocol()
```

Comment:
I'd strongly recommend writing these `Protocol` and `Well` objects as pytest fixtures, just to cutdown on rewriting boilerplate

a good example can be found in [TestTransferVolume](https://github.com/autoprotocol/autoprotocol-python/blob/3d09f1271e3695a260b0ce4e85a8445a3ef38b8e/test/protocol_test.py#L3258)

Reply:
Updated `provision_test.py` tests to use `pytest fixtures`